### PR TITLE
Fixes link styles for research and how-to articles

### LINKED ISSUE #1846: [bug] Change inline link colour
Issue:
Inline links (maybe all links) seem to be set to browser default colours which looks.. well.. defaulty. 
Also the "visited" state to purple looks so unloved :)

Link:
https://community.preciousplastic.com/

Screenshots:
Screenshot_2022-07-23_at_15_18_22

Proposal:
we set links to {
color: grey;
text-decoration: underline;
}
 
:hover {
underline: none;
}
:visited {
color: grey;
text-decoration: underline;
}

Screenshot 2022-07-23 at 15 19 47

### PR DESCRIPTION
**Pull request by asheerrizvi**
PR Checklist

- [x] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)

PR Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Developer experience (improves developer workflows for contributing to the project)

## Description

Link styles for research & how-to articles.

## Git Issues

Closes #1846 

## Screenshots/Videos

_If useful, provide screenshot or capture to highlight main changes_

---

## What happens next?

Thanks for the contribution! We try to make sure all PRs are reviewed ahead of a monthly dev call (first Monday of the month, open to all!).

If the PR is working as intended it'll be merged and included in the next platform release, if not changes will be requested and re-reviewed once updated.

If you need more immediate feedback you can try reaching out on slack in the `platform-dev` channel.

### CODE REVIEW

File: src/pages/Howto/Content/Howto/HowtoDescription/HowtoDescription.tsx [hunk 1]
```diff
@@ -242,7 +242,15 @@ export default class HowtoDescription extends PureComponent<IProps, IState> {
...
-              <Linkify properties={{ target: '_blank' }}>
+              <Linkify
+                properties={{
+                  target: '_blank',
+                  style: {
+                    color: theme.colors.grey,
+                    textDecoration: 'underline',
+                  },
+                }}
+              >
```
Comment (asheerrizvi) [inline src/pages/Howto/Content/Howto/HowtoDescription/HowtoDescription.tsx:253]: These components use a library called react-linkify (https://github.com/tasti/react-linkify) which is checking the markdown and converting any links which it is encountering to anchor tags. I was able to add the default styling (gray color and underline) but I am not able to specify hover styling. It's not documented in their docs as to how we can do that. I am not sure how we can do the hover styling here.

Comment (asheerrizvi) [reply src/pages/Howto/Content/Howto/HowtoDescription/HowtoDescription.tsx:253]: @thisismattia @davehakkens

**Comment by thisismattia** [comment]
hey @asheerrizvi thanks man 🤘 this works as expected 💪
As you're at it could you please add an hover state where the underline gets removed?

Big up!

File: src/pages/Howto/Content/Howto/HowtoDescription/HowtoDescription.tsx (hunk 1 above)
Comment (asheerrizvi) [reply src/pages/Howto/Content/Howto/HowtoDescription/HowtoDescription.tsx]: Was able to fix this

**Comment by thisismattia** [comment]
Thanks @asheerrizvi for finding a solution to the :hover state. Behaves nicely. Thank you ✌️
Over to the maintainers to merge :)

**Review by thisislawatts — APPROVED** [review]

**Pull request merged**